.github/workflows/release-build-zip-file.yml

```diff
@@ -94,65 +100,67 @@ jobs:
   verify-release-versions:
     name: 'Verify release and stable tag versions'
     runs-on: ubuntu-latest
-    if: ${{ inputs.skip_verify == 'false' }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch || github.ref }}
+          path: checkout-branch
```

Comment:
Can be a bit faster with e.g. (looks like we need the single file from source code):

```
sparse-checkout: |
  plugins/woocommerce/woocommerce.php
```

Reply:
Done.